### PR TITLE
Unregister Popup Confirmation 

### PR DESCRIPTION
**Pull request by bphoebew**

### CODE REVIEW

File: src/pages/events/[eventId]/EventUnRegisterModal.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
```
Comment (colin4554) [inline src/pages/events/[eventId]/EventUnRegisterModal.jsx]: This file should be under the components folder. Right now it is being treated like a page.

File: src/pages/events/[eventId]/index.js [hunk 2]
```diff
@@ -216,7 +217,7 @@ const EventInfo = () => {
               event.volunteers.includes(user._id) &&
               futureorTodaysDate && (
                 <Styled.Routing onClick={() => onUnregisterClicked(event)}>
-                  Unregister
```
Comment (colin4554) [inline src/pages/events/[eventId]/index.js]: I would leave this text as is

File: src/pages/events/[eventId]/EventUnRegisterModal.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <Modal isOpen={open} toggle={toggle} backdrop="static">
+      <ModalHeader toggle={toggle}>Delete Entry</ModalHeader>
+      <ModalBody>
+        Are you sure you want to cancel your registration for this event?
+      </ModalBody>
+      <ModalFooter>
+        <Button color="secondary" onClick={toggle}>
+          No, keep it
+        </Button>
+        <Button color="primary" onClick={handleSubmit} disabled={isDeleting}>
```
Comment (colin4554) [inline src/pages/events/[eventId]/EventUnRegisterModal.jsx]: Please update all the variables like isDeleting and text like Delete Entry so they make sense for the unregister flow

**Review by colin4554 — CHANGES_REQUESTED** [review]

**Comment by YiQin93** [comment]
This looks fantastic!

**Review by colin4554 — APPROVED** [review]

**Pull request merged**